frontend/src/styles/App.scss

```diff
@@ -9,12 +9,48 @@ body {
 .pmo-app {
     color: $text-color;
     display: flex;
+    flex: 1;
     /*flex-flow: nowrap;*/
-    margin: 0 auto;
-    width: 100%;
-    height: 100%;
+    //margin: 0 auto;
+    //max-width: 100%;
+    //height: 100%;
 }
 
-.main-section {
+main {
+    padding: 1rem;
+    display: flex;
+    > * {
+        width: 50%;
+    }
+
+}
+
+.debug {
```

Comment:
@ShawnTurple Is there a reason you didn't create an index.scss file alongside the Debug component and put its scss there? I get that this component may be a special circumstance, but I don't think we should be adding all our scss in one big file.

We should soon decide on how we're going to structure out our scss in this project but my vote would be for each component to live with its scss and that only more "global" scss would go in App.scss. Thoughts, @CR-gbc @ASpiteri-BCGov ?

Reply:
maybe old habits, also just seemed easier to do it this way, as it was a general debug style, and not a component specific style.

Reply:
What do you mean by "general debug style?" What I'm seeing is a .debug element that will have a table and an element named ".keycloak-token" as well. I don't see how this is a general pattern. To me, this describes a specific component, with a specific structure.

Again, I know this debug component is something of an outlier but I would discourage us from using App.scss as a dumping ground and having an intentional way to structure our scss. Adding component-specific scss is as simple as creating an adjacent "index.scss" file in a component folder and including it.

Reply:
i will update the debug, for its own scss.